src/pages/cli/auth/import.mdx

```diff
@@ -37,6 +37,14 @@ import attributesCallout from "/src/fragments/common/writable-vs-mutable-attribu
 
 <Fragments fragments={{all: attributesCallout}} />
 
+<Callout>
+
+Ensure that the hosted UI for a app client has a sign-out URL defined as omitting this may cause the Amplify CLI to not generate the OAuth `scopes`, `redirectSignIn`, `redirectSignOut` and `responseType` in `aws-exports.js` file. 
```

Comment:
```suggestion
Ensure that the hosted UI for an app client has a sign-out URL defined as omitting this may cause the Amplify CLI to not generate the OAuth `scopes`, `redirectSignIn`, `redirectSignOut` and `responseType` in the `aws-exports.js` file. 
```